src/main/java/uk/gov/hmcts/reform/unspec/service/flowstate/FlowPredicate.java

```diff
@@ -37,7 +37,7 @@
             && caseData.getApplicant1DefenceResponseDocument() != null;
 
     public static final Predicate<CaseData> schedulerStayClaim = caseData ->
-        caseData.getCcdState() == STAYED;
+        caseData.getCcdState() == STAYED && caseData.getDeemedServiceDateToRespondentSolicitor1() == null;
```

Comment:
should we use the caseData filed that we populate in full defence rather than confirm service?